test/public/CREATE/CreateX.deployCreate.t.sol

```diff
@@ -85,11 +40,11 @@ contract CreateX_DeployCreate_Public_Test is BaseTest {
 
         // We also check for the ERC-20 standard `Transfer` event.
         vm.expectEmit(true, true, true, true, computedAddress);
-        emit Transfer(zeroAddress, arg3, arg4);
+        emit IERC20.Transfer(zeroAddress, arg3, arg4);
```

Comment:
First time seeing this syntax in use, I like it

Reply:
Yeah that's possible since `0.8.21`, but due to an ICE of solc only really possible since `0.8.22`